install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -923,9 +923,8 @@ data:
   operator-api-serve-addr: {{ $defaultOperatorApiServeAddr | quote }}
 {{- end }}
 
-{{- if .Values.hubble.enabled }}
-  # Enable Hubble gRPC service.
   enable-hubble: {{ .Values.hubble.enabled | quote }}
```

Comment:
FYI there's some stray leading whitespace here.

Reply:
You mean the couple of spaces before `enable-hubble`? If yes, they are necessary to be under the top-level `data:` YAML block correct?

Reply:
Of course, sorry for the noise. :stuck_out_tongue: